nntrainer/models/neuralnet.cpp

```diff
@@ -591,45 +613,57 @@ int NeuralNetwork::train_run() {
   auto &label = last_layer_node->getOutputGrad(0);
   auto &in = first_layer_node->getInput(0);
 
+  /// below constant is needed after changing
+  /// databuffer having train, valid, test -> train buffer, valid buffer, test
+  /// buffer After the cahgne, only data train is used inside a databuffer.
+  /// RUN_CONSTANT is a stub value to deal with the situation
+  auto RUN_CONSTANT = DatasetDataUsageType::DATA_TRAIN;
```

Comment:
RUN_CONSTANT is temporary right, and will be removed once databuffer internally removed its dependency on the type the buffer it is?

Reply:
Correct ✔️